presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1031,6 +1031,404 @@ public void testVirtualBucketing()
         }
     }
 
+    // TODO: plan verification https://github.com/prestodb/presto/issues/16031
+    // @Test
```

Comment:
The right way to disable a test is by having:
```java
@Test(enabled = false)
```

Same for all other places in this class